demos/_includes/DemoActionsUtil.php

```diff
@@ -82,8 +84,8 @@ public static function setupDemoActions(CountryLock $country): void
             'edit_iso',
             [
                 'caption' => 'Edit ISO3',
-                'description' => function () {
-                    return 'Edit ISO3 for country:';
+                'description' => function (UserAction $action) {
+                    return 'Edit ISO3 for country: ' . $action->getEntity()->getTitle();
```

Comment:
No need to put it back since it will always be empty. This evaluate prior to entity being set.

Reply:
I noticed that. It is easy fixable?

Reply:
Dialog title is set when Modal is added to app->html which is always done prior to executing the action I believe. 
Perhaps in Modal Executor we could run javascript for changing Modal title when action is execute after entity is set.
As of now we can only use it to display Action or Model information only; not entity.

Reply:
> Dialog title is set when Modal is added to app->html which is always done prior to executing the action I believe.

Then this must be addressed, no JS please. I will keep the code with "empty entity title" so it can be fixed later...

I also think we can make `getTitle` throw easily if entity is passed, but unloaded :)